### DISCUSSION TITLE
Hi everyone, I would like to talk to you about running multiple pipeline versions in Kubernetes.

### DISCUSSION THREAD

**Discussion by yangxiao1220** [nextflow-io/nextflow#3237] [Q&A]
Currently I have v1.1 v1.2 v1.3 in my code repository, there may be more in the future, and these versions may run in parallel on the same cluster, the former will be overwritten by the latter version, because they share A code storage directory. And I found that nextflow always reads the latest nextflow.config of the code repository, I think it should read the nextflow.conf of the specified running version

**Comment by mribeirodantas** [comment]
You should not run at the same time nextflow more than once in the same directory. You could call nextflow at the same time from different directories, though.

```
mkdir git_branch1_here
cd git_branch1_here
nextflow run nextflow-io/rnaseq -r git_branch1_here
```

Then in another terminal:
```
mkdir git_branch2_here
cd git_branch2_here
nextflow run nextflow-io/rnaseq -r git_branch2_here
```

**Reply by yangxiao1220** [reply]
Hi thanks for your reply, I'm currently running my task with `nextflow kuberun`, so I'm facing this problem, as you said if you post in sge or slurm you can segregate different versions with a directory

**Reply by mribeirodantas** [reply]
You can provide a different configuration file for a pipeline run with `-C <configuration file here>`. Do you think it works for your situation?
